cli/Valet/Brew.php

```diff
@@ -36,6 +36,26 @@ function __construct(CommandLine $cli, Filesystem $files)
     {
         $this->cli = $cli;
         $this->files = $files;
+        $this->homebrew_prefix = '/usr/local';
+        $this->prefix_loaded = false;
+    }
+
+    /**
+     * Determine HOMEBREW_PREFIX
```

Comment:
In your modified environment, does calling `brew --prefix` work properly as a replacement for this custom function?

Reply:
OMG. yep.  just use that!! "brew --prefix"

Reply:
I modified the code to use "brew --prefix" internally. It was a better choice.
Internally brew uses cwd, so getting the path from which brew was more like what it does - but as it has a switch, we should use it.